src/pages/editor/koni/components/EditorDetailPanel/DetailForm.tsx

```diff
@@ -0,0 +1,137 @@
+import { Form } from '@ant-design/compatible';
+import { Card, Input, Select } from 'antd';
+import React, { Fragment } from 'react';
+
+import { FormComponentProps } from 'antd/es/form';
+import { withPropsAPI } from 'gg-editor';
+
+const upperFirst = (str: string) =>
+  str.toLowerCase().replace(/( |^)[a-z]/g, (l: string) => l.toUpperCase());
+
+const { Item } = Form;
+const { Option } = Select;
+
+const inlineFormItemLayout = {
+  labelCol: {
+    sm: { span: 8 },
+  },
+  wrapperCol: {
+    sm: { span: 16 },
+  },
+};
+
+interface DetailFormProps extends FormComponentProps {
+  type: string;
+  propsAPI?: any;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.](https://app.codacy.com/manual/afc163/ant-design-pro/pullRequest?prid=4583897)